script/events/room.js

```diff
@@ -645,6 +651,43 @@ Events.Room = [
 					}
 				}
 			},
+			'killman':{
+				text: [
+					_("beaten to a pulp"),
+					_("the man had very little"),
+					_("you see his family in the distance"),
```

Comment:
ADR doesn't use personal pronouns https://github.com/doublespeakgames/adarkroom/wiki/Design-Guide

Try something like 'others cower in the distance'